web_toggle/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Boolean Bootstrap Toggle
+========================
+
+This widget provides more options on displaying a boolean field within Odoo's backend form views. Instead of 
+'Checked' or 'Not Checked' you can now use this widget to display a Switch Button which will slide from 'On' to 
+'Off' or 'Yes' to 'No' or whatever you have configured in your options. 
+
+This widget is capable of utilizing all of bootstrap toggle's options including varrying colours, sizes, text and 
+shape of the toggle widget. For instance you could display a "Yes" in Green and "No" in Red to contrast the two 
+options.
+
+Usage
+=====
+
+After you installed it, you can add the text widget='boolean_switch' to any boolean within a form view. This 
+will change the checkbox to an 'On' or 'Off' button which slide from one to the other upon clicking. There are 
+many options within the bootstrap-toggle library which you can use to make additional changes. They can all be 
+utilized by providing an options dictionary. Go to bootstrap-toggle documentation to learn about all available 
+options such as colours, size, text, shapes, etc. Here is a full example usage.
+
+<field name="my_boolean" widget='boolean_switch' options="{'data-on':'Yes','data-off':'No','data-onstyle':'primary','data-offstyle':'danger','data-size':'mini'}"/>
```

Comment:
It would be nice to format this [as code](http://sphinx-doc.org/rest.html#source-code), but I won't pick on that.